kiwi/boot/image/builtin_kiwi.py

```diff
@@ -167,5 +167,7 @@ def create_initrd(self, mbrid=None):
             compress = Compress(
                 os.sep.join([self.target_dir, self.initrd_base_name])
             )
-            compress.xz(self.xz_options)
+            compress.xz(
+                ['--check=crc32', '--lzma2=dict=1MiB', '--threads=0']
+            )
```

Comment:
Right, so we force the initrd compression options, fine with me, is safer if users can't mess with it.